### ISSUE TITLE
Please tag version of gsplat==0.1.8

### ISSUE THREAD

**Issue by ichsan2895** [nerfstudio-project/gsplat#141]
There are important fix by @jb-ye , so the version is `0.1.8`
![image](https://github.com/nerfstudio-project/gsplat/assets/60387342/ed754a4f-70a4-4c0e-a96d-4df4475ec48c)

But, When I installed `gsplat` for first time by installing nerfstudio
```
git clone https://github.com/nerfstudio-project/nerfstudio
cd nerfstudio/
pip install -e .
```

I get `gsplat==0.1.7` which I know when I ran `pip list`.
![image](https://github.com/nerfstudio-project/gsplat/assets/60387342/4ef98aee-13a0-449e-b070-99d313741755)

So please fix gsplat to version 0.1.8 since the `gsplat==0.1.7` has problem which reduce quality significantly

**Comment by ichsan2895** [comment]
Proof the reduction of quality of `gsplat==0.1.7`, it already fixed in 0.1.8. But I dont get `gsplat==0.1.8` when I installed nerfstudio.
![image](https://github.com/nerfstudio-project/gsplat/assets/60387342/8036a77a-6b51-4d46-97a1-540f2fac2af7)

The way to fix it is installed manually compile from source`gsplat` by run this command `pip install git+https://github.com/nerfstudio-project/gsplat.git`

**Comment by ichsan2895** [comment]
Closed as gsplat==0.1.8 already released
